contracts/VeJoeStaking.sol

```diff
@@ -262,98 +165,65 @@ contract VeJoeStaking is Initializable, OwnableUpgradeable {
     /// @notice Claim any pending veJOE
     function claim() external {
         require(_getUserHasNonZeroBalance(msg.sender), "VeJoeStaking: cannot claim veJOE when no JOE is staked");
+        updateRewardVars();
         _claim();
     }
 
     /// @notice Get the pending amount of veJOE for a given user
     /// @param _user The user to lookup
-    /// @return The number of pending veJOE tokens for `_user`
-    function getPendingVeJoe(address _user) public view returns (uint256) {
+    /// @return pendingVeJoe The number of pending veJOE tokens for `_user`
+    function getPendingVeJoe(address _user) public view returns (uint256 pendingVeJoe) {
```

Comment:
Any reason why you choose to declare the return variable here?

Reply:
Indeed better not to, as we don't always return `pendingVeJoe`